awspub/image.py

```diff
@@ -353,6 +354,19 @@ def _public(self) -> None:
             else:
                 logger.error(f"image {self.image_name} not available in region {region}. can not make public")
 
+    def _sns_publish(self) -> None:
+        """
+        Publish SNS notifiations about newly available images to subscribers
+        """
+        for region in self.image_regions:
+            ec2client_region: EC2Client = boto3.client("ec2", region_name=region)
+            image_info: Optional[_ImageInfo] = self._get(ec2client_region)
+
+            if not image_info:
+                logger.error(f"can not send SNS notification for {self.image_name} because no image found in {region}")
+                return
+            SNSNotification(self._ctx, self.image_name, region).publish()
```

Comment:
what would this look like as a consumer of this message? are we sending the exact same text for every region? how do I advertise the `image_name` and `ami_id` inside the message, or do we care about that? (e.g. for our use case i could see us doing a generic message that is use this ssm param path for finding the ID).

Reply:
currently there's no way to advertise the `ami_id` (we could hardcode the image name or ssm path in the message via the config.yaml file).
Jess and I talked about the `ami_id` and said we'll keep that out for now. 
The `ami_id` is "dynamic" information so we would need to introduce some sort of templating into the message. I would like to avoid that for now.